### ISSUE TITLE
flutter run fails with iOS 10 devices

### ISSUE THREAD

**Issue by cbracken** [flutter/flutter#6027]
## Steps to Reproduce
In the hello_world sample:
```shell
flutter run
```

Results in the following console output:
```
Exception from flutter run: SocketException: OS Error: Connection refused, errno = 61, address = 127.0.0.1, port = 49582
```

**Comment by cbracken** [comment]
The underlying issue appears to be a change to the log format.

With an iOS 10 device I see:
```
Sep 23 15:52:02 cbracken-iPhone Runner(libsystem_asl.dylib)[6166] <Info>: Observatory listening on http://127.0.0.1:54951
```

We currently scrape logs with the following regex:
```
Runner\[[\d]+\] <[A-Za-z]+>:
```

**Comment by sethladd** [comment]
Good find. Thanks for testing this!

**Comment by cbracken** [comment]
Fixed by #6028
